src/Common/src/System/Collections/HashHelpers.cs

```diff
@@ -40,24 +30,41 @@ internal static class HashHelpers
             1103, 1327, 1597, 1931, 2333, 2801, 3371, 4049, 4861, 5839, 7013, 8419, 10103, 12143, 14591,
```

Comment:
I would add a comment explaining why we don't just grow the table... since apparently it wasn't obvious to us

Reply:
added.